tests/data_context/migrator/test_cloud_migrator.py

```diff
@@ -53,6 +50,53 @@ def test__send_configuration_bundle_sends_valid_http_request(
     )
 
 
+@pytest.mark.unit
+@pytest.mark.cloud
+def test__send_validation_results_sends_valid_http_request():
```

Comment:
I'm assuming tests to make sure logging messages are printed will be in a separate PR? (that is fine just want to make sure they don't get missed)

Reply:
yup!